packages/dui3/store/accounts.ts

```diff
@@ -1,10 +1,169 @@
 import { defineStore } from 'pinia'
-import { useInjectedAccounts } from '~/lib/accounts/composables/setup'
+import type { ApolloLink } from '@apollo/client/core'
+import { ApolloClient, InMemoryCache, gql, HttpLink, split } from '@apollo/client/core'
+import { ApolloClients, provideApolloClients } from '@vue/apollo-composable'
+import type { Account } from '~/lib/bindings/definitions/IAccountBinding'
+import { WebSocketLink } from '@apollo/client/link/ws'
+import { getMainDefinition } from '@apollo/client/utilities'
+import { setContext } from '@apollo/client/link/context'
+
+export type DUIAccount = {
+  /** account info coming from the host app */
+  accountInfo: Account
+  /** the graphql client; a bit superflous */
+  client: ApolloClient<unknown>
+  /** whether an intial serverinfo query succeeded. */
+  isValid: boolean
+}
+
+const accountTestQuery = gql`
```

Comment:
DUI3 has graphql-codegen, make sure you use the proper gql helper to define all operations so that you get typed results: `import { graphql } from '~~/lib/common/generated/gql'`